package.json

```diff
@@ -109,7 +109,6 @@
     "predocs-test": "npm run docs",
     "prepare": "npm run compile-protos && npm run compile",
     "prelint": "cd samples; npm link ../; npm install",
-    "clean": "gts clean",
```

Comment:
why did you remove this script?

Reply:
My git system automatically deleted it.
So, I see that "clean" command using twice in package.json for now.